docs/root/version_history/current.rst

```diff
@@ -30,6 +30,7 @@ Removed Config or Runtime
 
 * http: removed `envoy.reloadable_features.allow_500_after_100` runtime guard and the legacy code path.
 * http: removed `envoy.reloadable_features.hcm_stream_error_on_invalid_message` for disabling closing HTTP/1.1 connections on error. Connection-closing can still be disabled by setting the HTTP/1 configuration :ref:`override_stream_error_on_invalid_http_message <envoy_v3_api_field_config.core.v3.Http1ProtocolOptions.override_stream_error_on_invalid_http_message>`.
+* http: removed `envoy.reloadable_features.http_match_on_all_headers` runtime guard and legacy code paths.
```

Comment:
this will need to rebase - `inline literals` now need to be double backticked (which is the semantically correct way in rst)

im in the process of updating the guidance (and improving the rendering)...

Reply:
Done - thanks for the heads up!